### PR TITLE
Add an ROADMAP for Hub

### PR DESCRIPTION
**Pull request by piyush-garg**
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ - It should warn user in case of mismatch with supported Pipeline version of resource
+
+### Automate Hub Deployment
+
+ - Hub deployment should be automated through script.
+
+### Catlin Integration
+
+ - Catlin should be integrated with catalog CI to validate resource according to TEP
+ - Catlin should support both Task and Pipeline TEP proposal
```
Comment (vdemeester) [inline ROADMAP.md:36]: This isn't *really* related to Hub, but our catalog, am I right ?

**Review by vdemeester — COMMENTED** [review]
Do we want to add something like "Self-hosting the Hub", aka having some scripts, … in releases so that users can install their version of the hub on their infrastructure if need be ?

**Review by vdemeester — APPROVED** [review]

**Comment by piyush-garg** [comment]
@vdemeester @sm43 @PuneetPunamiya @vinamra28 @pratap0007 

Ready to review

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+### Multi Catalog Support
+
+ - User should be able to provide multi catalogs in config
+ - API's should be able to act for multiple catalogs
+ - UI should show the resources from all catalogs mentioned and also able to filter based on catalog
+ - CLI should be able to work with multiple catalogs
+
+### Add support for Pipeline Kind
+
+ - Hub should the Pipelines Kind resources
```
Comment (PuneetPunamiya) [inline ROADMAP.md:14]: nit:
```suggestion
 - Hub should support the Pipelines Kind resources
```

**Comment by sm43** [comment]
lgtm

File: ROADMAP.md (hunk 2 above)
Comment (piyush-garg) [reply ROADMAP.md]: resolved

File: ROADMAP.md (hunk 1 above)
Comment (piyush-garg) [reply ROADMAP.md]: removed from RoadMap

**Pull request merged**